### ISSUE TITLE
Problem with android compression

### ISSUE THREAD

**Issue by mannol** [shahen94/react-native-video-processing#89]
When `width` and `height` props are used as `compress` options in `ProcessingManager.compress()`, they are parsed with decimal points to scale parameter which causes compression to fail. 

The following code in Trimmer.java produces: `Copmress resolution: -> 405.0:720.0` when it should be `Copmress resolution: -> 405:720`. 
```JAVA
if (width != null && height != null) {
  Log.w(LOG_TAG, "Copmress resolution: -> " + Double.toString(width) + ":" + Double.toString(height));
  cmd.add("-vf");
  cmd.add("scale=" + Double.toString(width) + ":" + Double.toString(height));
}
```

`Double.toString()` simply won't cut it.

**Comment by kesha-antonov** [comment]
Nice catch!

**Comment by kesha-antonov** [comment]
Hey @mannol 
Did you encountered issue when video wasn't trimmed? I mean you try to trim video but it remains the same. In what situations that could be?

**Comment by mannol** [comment]
@kesha-antonov no. Didn't have any issues with trimming other than it too was a blocking operation.

**Comment by kesha-antonov** [comment]
Ok. try this https://github.com/shahen94/react-native-video-processing/pull/91

**Comment by kesha-antonov** [comment]
Sorry for spamming this thread.

@mannol What do you think - maybe we should do "copy" and "chmod" async task too? https://github.com/shahen94/react-native-video-processing/blob/master/android/src/main/java/com/shahenlibrary/Trimmer/Trimmer.java#L540

I'm worried that when user opens app first he'll wait until ffmpeg will be copied and chmod'ed

**Comment by mannol** [comment]
Yeah. Also, that should be done once only if the file doesn't exist already.

**Comment by kesha-antonov** [comment]
There is check already

**Comment by kesha-antonov** [comment]
Ok. I'll add async task for that later

**Comment by mannol** [comment]
Great!

**Comment by gvillenave** [comment]
@mannol @kesha-antonov Check out #126 for a fix for the original issue here.
